demo/src/app/components/modal/modal.component.ts

```diff
@@ -0,0 +1,12 @@
+import {Component} from '@angular/core';
+import {NgbdApiDocs} from '../shared/api-docs';
+
+@Component({
+  selector: 'ngbd-modal',
+  template: `
+    <ngbd-api-docs directive="NgbModalWindow"></ngbd-api-docs>
+    <ngbd-api-docs directive="NgbModalBackdrop"></ngbd-api-docs>
```

Comment:
Wait, so what am I missing?  Where is NgbModal?  Is it not done yet because of ng2 blockers?